scripts/app-image-support/AppRun

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
nit: use the same spelling everywhere for appimage-related files, "appimage" not "app-image". Also the directory can just be named `appimage`, "support" is redundant.

Reply:
Renamed.